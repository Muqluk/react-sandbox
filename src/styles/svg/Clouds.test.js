import React from 'react';
import { shallow } from 'enzyme';
import renderTests from '~/test-helpers';
import CloudsSvg from './Clouds';

function generateWrapper(props) {
  return shallow(<CloudsSvg {...props} />);
}

describe('CloudsSvg', () => {
  renderTests(() => generateWrapper());
});
