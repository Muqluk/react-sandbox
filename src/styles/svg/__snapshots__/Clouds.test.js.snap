// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CloudsSvg renders and matches snapshot 1`] = `
<svg
  className="cc-clouds-svg"
  data-name="Layer 1"
  id="Layer_1"
  viewBox="0 0 305.77 237.76"
  xmlns="http://www.w3.org/2000/svg"
>
  <title>
    Clouds
  </title>
  <path
    className="cls-1"
    d="M211.72,19.13s-40.19-37.2-71.38,18.59c0,0-52.79-.2-52.79,31.8,0,25.79,22.2,31.19,22.2,31.19S98.94,152.9,144.54,150.5s40.19-4.8,40.19-4.8,21.59,29.38,50.39,19.8,26.39-39.6,26.39-39.6,37.19-11.4,33.59-33-33.59-31.2-33.59-31.2,19.19-18.6,3-36S211.72,19.13,211.72,19.13Z"
    data-name="Path 837"
    id="Path_837"
  />
  <path
    className="cls-1"
    d="M75.42,149.38s-25.15-17.47-38.09,15.85c0,0-29.24,4-26.75,21.72,2,14.28,14.72,15.54,14.72,15.54s-1.92,29.74,23.14,24.86,21.88-5.78,21.88-5.78,14.24,14.59,29.44,7,11.53-24,11.53-24,19.71-9.2,16-20.88-21-14.66-21-14.66,9.18-11.79-1.14-20.17S75.42,149.38,75.42,149.38Z"
    data-name="Path 839"
    id="Path_839"
  />
</svg>
`;
